.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
+
+*.sh text eol=lf
```

Comment:
this will enforce lf, right? Actually I'd like to keep the previous setting, because this setting will show all files as changed, but they are not.